docs/css/extra.css

```diff
@@ -0,0 +1,104 @@
+
+:root {
+    /* Auto Trader variables with colours taken from https://github.atcloud.io/pages/AutoTrader/sparkplug/components/detail/colours.html */
```

Comment:
We should maybe remove this as the source will be public

Reply:
The action should generate the CNAME for me